schema/show.js

```diff
@@ -34,9 +34,11 @@ import {
   GraphQLUnionType,
 } from 'graphql';
 
-const kind = ({ artists, fair, artists_without_artworks }) => {
+const kind = ({ artists, fair, artists_without_artworks, group }) => {
   if (isExisty(fair)) return 'fair';
-  if (artists.length > 1 || (artists_without_artworks && artists_without_artworks.length > 1)) {
+  if (group ||
```

Comment:
It's less pretty, but it's more helpful when ralearon the code on e.g. GitHub, which has a column width of I think 120.